inst/easystats_bib.bib

```diff
@@ -0,0 +1,241 @@
+  version = {%s}
+}
+
+@article{correlationArticle,
+  title = {Methods and Algorithms for Correlation Analysis in {{R}}},
+  author = {Makowski, Dominique and Ben-Shachar, Mattan and Patil, Indrajeet and L\"udecke, Daniel},
+  date = {2020-07-16},
+  journaltitle = {Journal of Open Source Software},
+  shortjournal = {JOSS},
+  volume = {5},
+  number = {51},
+  pages = {2306},
+  issn = {2475-9066},
+  doi = {10.21105/joss.02306},
+  url = {https://joss.theoj.org/papers/10.21105/joss.02306},
+  urldate = {2022-03-23}
+}
+
+@software{correlationPackage,
+  title = {{{correlation}}: Methods for Correlation Analysis},
+  shorttitle = {{{correlation}}},
+  author = {Makowski, Dominique and Wiernik, Brenton M. and Patil, Indrajeet and L\"udecke, Daniel and Ben-Shachar, Mattan S.},
+  date = {2022-02-14},
+  origdate = {2020-03-16},
+  url = {https://CRAN.R-project.org/package=correlation},
+  urldate = {2022-03-23},
+  abstract = {Lightweight package for computing different kinds of correlations, such as partial correlations, Bayesian correlations, multilevel correlations, polychoric correlations, biweight correlations, distance correlations and more. Part of the 'easystats' ecosystem.},
+  version = {%s}
+}
+
+@software{datawizardPackage,
+  title = {{{datawizard}}: Easy Data Wrangling},
+  shorttitle = {{{datawizard}}},
+  author = {Makowski, Dominique and L\"udecke, Daniel and Patil, Indrajeet and Ben-Shachar, Mattan S. and Wiernik, Brenton M.},
+  date = {2022-03-03},
+  origdate = {2021-06-18},
+  url = {https://CRAN.R-project.org/package=datawizard},
+  urldate = {2022-03-28},
+  abstract = {A lightweight package to easily manipulate, clean, transform, and prepare your data for analysis. It also forms the data wrangling backend for the packages in the 'easystats' ecosystem.},
+  version = {%s}
+}
+
+@software{easystatsPackage,
+  title = {{{easystats}}: Streamline Model Interpretation, Visualization, and Reporting},
+  shorttitle = {{{easystats}}},
+  author = {L\"udecke, Daniel and Makowski, Dominique and Ben-Shachar, Mattan S. and Patil, Indrajeet and Wiernik, Brenton M.},
+  date = {2022-03-27T22:06:06Z},
+  origdate = {2019-01-28T10:39:29Z},
+  url = {https://github.com/easystats/easystats},
+  urldate = {2022-03-28},
+  version = {%s}
+}
+
+@article{effectsizeArticle,
+  title = {{{effectsize}}: Estimation of Effect Size Indices and Standardized Parameters},
+  shorttitle = {{{effectsize}}},
+  author = {Ben-Shachar, Mattan S. and L\"udecke, Daniel and Makowski, Dominique},
+  date = {2020-12-23},
+  journaltitle = {Journal of Open Source Software},
+  shortjournal = {JOSS},
+  volume = {5},
+  number = {56},
+  pages = {2815},
+  issn = {2475-9066},
+  doi = {10.21105/joss.02815},
+  url = {https://joss.theoj.org/papers/10.21105/joss.02815},
+  urldate = {2021-06-30}
+}
+
+@software{effectsizePackage,
```

Comment:
I think this is incorrect. 

AFAIK, package citations include only authors, and not contributors, so it should not include IP and BW. 

For example, _broom_ has a ton of contributors, but only the three authors are included in the citation.

``` r
citation("broom")
#> 
#> To cite package 'broom' in publications use:
#> 
#>   David Robinson, Alex Hayes and Simon Couch (2022). broom: Convert
#>   Statistical Objects into Tidy Tibbles. R package version 0.7.12.
#>   https://CRAN.R-project.org/package=broom
#> 
#> A BibTeX entry for LaTeX users is
#> 
#>   @Manual{,
#>     title = {broom: Convert Statistical Objects into Tidy Tibbles},
#>     author = {David Robinson and Alex Hayes and Simon Couch},
#>     year = {2022},
#>     note = {R package version 0.7.12},
#>     url = {https://CRAN.R-project.org/package=broom},
#>   }
```

<sup>Created on 2022-03-29 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

Reply:
https://github.com/easystats/effectsize/pull/433 seems appropriate?

Reply:
Yeah, that's ok.

Reply:
Yup.
Should we go ahead and have the 5 of us as aut on all packages? Is that already the case?

Reply:
I thought it was the case, so I think effectsize might be the only standout, but we should double check